k8s/sample-app/README.md

```diff
@@ -138,14 +138,27 @@ script:
 export IMAGE_SAMPLE_APP=$(docker pull $IMAGE_SAMPLE_APP | awk -F: "/^Digest:/ {print gensub(\":.*$\", \"\", 1, \"$IMAGE_SAMPLE_APP\")\"@sha256:\"\$3}")
 ```
 
+For the persistent disk used by the sample-app deployment, you will need to:
+
+ * Set the StorageClass name. Check your available options using the command below:
+   * ```kubectl get storageclass```
+   * Or check how to create a new StorageClass in [Kubernetes Documentation](https://kubernetes.io/docs/concepts/storage/storage-classes/#the-storageclass-resource)
+
+ * Set the persistent disk's size. The default disk size is "8Gi".
+
+```shell
+export DEFAULT_STORAGE_CLASS="standard" # provide your StorageClass name if not "standard"
```

Comment:
We should avoid hardcoding `standard` as the default because that could change in the future.

Is it possible to set `DEFAULT_STORAGE_CLASS` to empty string by default. And if it is set to empty string, do not even specify a `storageClassName: ...` field in the manifest (so the cluster default StorageClass is used.

Reply:
Hmm, good point. I think that would be a lot more easier to do if we are using some manifest management tool like Helm. Otherwise I think we might have to do something non-trivial to refactor the way manifest is rendered.